src/lib/analytics/types.ts

```diff
@@ -8,6 +8,8 @@ export type ScreenEvent = (
   properties?: ScreenPropertiesMap[keyof ScreenPropertiesMap],
 ) => Promise<void>
 interface TrackPropertiesMap {
+  // OS events
+  'OpenApp:FromNotification': {}
```

Comment:
I would probably call this

`Notifications:Click` or `Notificatons:OpenApp`

My reasoning is that in the future we will probably deep-link notifications to the exact reply, quote, etc. and then we would want to track that differently as well.

Reply:
Sure sg